support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx

```diff
@@ -1,4 +1,3 @@
-import * as React from 'react';
```

Comment:
This must have been missed by the codemod we used to upgrade to emotion 11 https://github.com/guardian/support-frontend/pull/3439